src/ui/Font.cpp

```diff
@@ -80,6 +135,65 @@ int Font::height() const {
 	}
 }
 
+int Font::textHeight(std::string str, int override_pointsize) const {
+	int numNewLines = std::count(str.begin(), str.end(), '\n');
+
+	return numNewLines * height(override_pointsize); //TODO: Do we need to account for a gap between every line??
```

Comment:
I'm having some fonts overlap (but others don't), which is very strange and a bit unsettling. Need to look into this more...